app/commands/submission/test_run/process.rb

```diff
@@ -3,17 +3,16 @@ class TestRun
     class Process
```

Comment:
This now pivots on where it was a head run or not as to how some of the behaviours work